staramr/detection/AMRDetection.py

```diff
@@ -90,6 +90,7 @@ def run_amr_detection(self, files, pid_threshold, plength_threshold_resfinder, p
 
         self._summary_dataframe = self._create_amr_summary(files, self._resfinder_dataframe, 
                                                            self._pointfinder_dataframe, self._plasmidfinder_dataframe)
+        self._summary_dataframe = self._summary_dataframe.fillna('None')
```

Comment:
This is mainly for the Plasmid Genes column since for some reason if you have a mix of files that have plasmids and no plasmids it doesn't show None for the Plasmid Genes column.

Reply:
Thank you for solving this issue.